interop-java/src/main/java/scalaz/zio/java/IO.java

```diff
@@ -0,0 +1,54 @@
+package scalaz.zio.java;
+
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+public class IO<E, A> {
+
+    private scalaz.zio.IO<E, A> zio;
```

Comment:
let it be delegate?

Reply:
You mean the variable be named "delegate" instead of "zio"?

Reply:
Yes,more cleaner.